lib/types/gpt.nix

```diff
@@ -129,11 +129,14 @@ in
     _config = lib.mkOption {
       internal = true;
       readOnly = true;
-      default = map
+      default = (map
           (partition:
             lib.optional (partition.content != null) partition.content._config
           )
-          (lib.attrValues config.partitions);
+          (lib.attrValues config.partitions))
+          ++ (lib.optional (lib.any (part: part.type == "EF02") (lib.attrValues config.partitions)) {
```

Comment:
We should mention this in our documentation that one should use EF02 to create a partition for grub.

Reply:
but where?

Reply:
https://github.com/nix-community/disko/blob/master/docs/quickstart.md